packages/components/organisms/f-content-cards/stories/content-cards/all-cards.js

```diff
@@ -0,0 +1,72 @@
+import {
+    ContentCards,
+    FirstTimeCustomerCard,
+    PostOrderCard,
+    PromotionCard, StampCard1,
+    TermsAndConditionsCard,
+    VoucherCard
+} from '../../src';
+import { allCardsMockSetup } from './story-utils/braze-mocks';
+import methods from './story-utils/methods';
+import { STATE_DEFAULT } from '../../src/components/ContentCards';
+
+const components = {
+    ContentCards,
+    VoucherCard,
+    FirstTimeCustomerCard,
+    PromotionCard,
+    TermsAndConditionsCard,
+    PostOrderCard,
+    StampCard1
+};
+
+export default function ContentCardsBraze (args, { argTypes }) {
+    return {
+        name: 'Content Cards Braze',
+
+        components,
+
+        props: Object.keys(argTypes),
+
+        methods,
+
+        /**
+         * Ensures that card mocks are set up
+         */
+        beforeCreate: allCardsMockSetup,
+
+        template: `
+<content-cards
```

Comment:
Indentation here is a bit off

Reply:
@dandel10n - I was keeping it so the top level element in the template was at column zero for each of these, as IMO it makes it easier for the reader to interpret the template, as no indentation level is prescribed by linting for templates given inline here. What indent rules should I be following here, in your opinion?

Reply:
I thought like this would look better: https://github.com/justeat/fozzie-components/blob/e0001773a4a12ec2686bb98fd90674e48106777b/packages/components/organisms/f-footer/stories/footer.stories.js#L34-L40

Reply:
Thanks - I'll action for this and the ones below.